public/app/features/alerting/unified/components/rule-editor/alert-rule-form/simplifiedRouting/route-settings/RouteSettings.tsx

```diff
@@ -42,12 +41,17 @@ export const RoutingSettings = ({ alertManager }: RoutingSettingsProps) => {
     formState: { errors },
   } = useFormContext<RuleFormValues>();
   const [groupByOptions, setGroupByOptions] = useState(stringsToSelectableValues([]));
-  const { groupBy, groupIntervalValue, groupWaitValue, repeatIntervalValue } = useGetDefaultsForRoutingSettings();
+  const { groupIntervalValue, groupWaitValue, repeatIntervalValue } = getDefaultsForRoutingSettings();
   const overrideGrouping = watch(`contactPoints.${alertManager}.overrideGrouping`);
   const overrideTimings = watch(`contactPoints.${alertManager}.overrideTimings`);
-  const requiredFieldsInGroupBy = ['grafana_folder', 'alertname'];
-  const disableGrouping = '...';
+  const requiredFieldsInGroupBy = useMemo(() => ['grafana_folder', 'alertname'], []);
   const styles = useStyles2(getStyles);
+  useEffect(() => {
+    if (overrideGrouping && watch(`contactPoints.${alertManager}.groupBy`)?.length === 0) {
```

Comment:
I think `getValues` could be a better option than `watch` 
We're not listening on `contactPoints.alertmanager.groupby` changes in this case but just want to get the current value, am I right?

Reply:
yep, I think you're right 👍